gstreamer-sdp/src/s_d_p_media.rs

```diff
@@ -22,6 +22,7 @@ use s_d_p_connection::SDPConnection;
 use s_d_p_key::SDPKey;
 
 glib_wrapper! {
+    #[repr(C)]
```

Comment:
This does not need `#[repr(C)]`. For functions to return a reference to an `SDPMedia`, you would use `from_glib_borrow()`